package.json

```diff
@@ -30,12 +30,43 @@
     "gulp-rev-delete-original": "^0.2.3",
     "gulp-stylus": "^2.7.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-webserver": "^0.9.1"
+    "gulp-webserver": "^0.9.1",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "xo": "^0.23.0"
   },
-  "author": "Swapnil Agarwal",
-  "license": "CC0-1.0",
-  "bugs": {
-    "url": "https://github.com/swapagarwal/swag-for-dev/issues"
+  "scripts": {
+    "lint": "xo",
+    "lint:fix": "xo --fix",
+    "build": "gulp build"
+  },
+  "xo": {
+    "rules": {
+      "unicorn/prefer-spread": "off"
+    },
+    "overrides": [
```

Comment:
Maybe we should do an inverse override, since only the gulpfile is not browser specific - default env is browser, and gulpfile is configured to be node